src/components/molecules/Visualizer/Widget/Timeline/index.tsx

```diff
@@ -0,0 +1,36 @@
+import TimelineUI from "@reearth/components/atoms/Timeline";
+import { ComponentProps as WidgetProps } from "@reearth/components/molecules/Visualizer/Widget";
+import { styled } from "@reearth/theme";
+
+import { useTimeline } from "./hooks";
+
+export type Props = WidgetProps;
+
+const Timeline = ({ widget, sceneProperty }: Props): JSX.Element | null => {
+  const { isOpened, currentTime, range, events } = useTimeline();
+
+  return (
+    <Widget extended={!!widget?.extended?.horizontally}>
+      <TimelineUI
+        isOpened={isOpened}
+        currentTime={currentTime}
+        range={range}
+        sceneProperty={sceneProperty}
+        {...events}
+      />
+    </Widget>
+  );
+};
+
+const Widget = styled.div<{
+  extended?: boolean;
+}>`
+  max-width: 100vw;
+  width: ${({ extended }) => (extended ? "100%" : "500px")};
+
+  @media (max-width: 560px) {
+    width: ${({ extended }) => (extended ? "100%" : "90vw")};
```

Comment:
Here too, please check with @lavalse

Reply:
According to him there is no need to change this style